hs_core/models.py

```diff
@@ -942,34 +938,29 @@ class Relation(AbstractMetaDataElement):
     """Define Relation custom metadata model."""
 
     SOURCE_TYPES = (
-        ('isHostedBy', 'The content of this resource is hosted by'),
-        ('isCopiedFrom', 'The content of this resource was copied from'),
-        ('isDataFor', 'The content of this resource serves as the data for'),
-        ('cites', 'This resource cites'),
-        ('isPartOf', 'The content of this resource is part of'),
-        ('hasPart', 'This resource includes'),
-        ('isExecutedBy', 'The content of this resource can be executed by'),
-        ('isCreatedBy', 'The content of this resource was created by a related App or software program'),
-        ('isVersionOf', 'This resource updates and replaces a previous version'),
-        ('isReplacedBy', 'This resource has been replaced by a newer version'),
-        ('isDescribedBy', 'This resource is described by'),
-        ('conformsTo', 'This resource conforms to established standard described by'),
-        ('hasFormat', 'This resource has a related resource in another format'),
-        ('isFormatOf', 'This resource is a different format of'),
-        ('isRequiredBy', 'This resource is required by'),
-        ('requires', 'This resource requires'),
-        ('isReferencedBy', 'This resource is referenced by'),
-        ('references', 'The content of this resource references'),
-        ('replaces', 'This resource replaces'),
-        ('source', 'The content of this resource is derived from')
+        (RelationTypes.isPartOf.value, 'The content of this resource is part of'),
```

Comment:
@pkdash ```RelationTypes.isPartOf``` is mapped to a string in enum. Then ```RelationTypes.isPartOf.value``` should not be valid, right?

Reply:
@hyi Initially, I had it as `RelationTypes.isPartOf`. But that showed up in the migration (https://github.com/hydroshare/hydroshare/pull/4468/files#diff-6a9a1d2b5eaaa6a2667503aa64d06c1048a2638744bbade4e67fe92b84f12b1fR26) setting `choices` for example for the `isPartOf` as `(hs_core.enums.RelationTrype.isPartOf('isPartOf'): 'The content of this resource is part of')` which I was not expecting. 

By using `RelationTypes.isPartOf.value` the `choices` in the migration is now set like: `('isPartOf': 'The content of this resource is part of')`